src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingAttributes.cs

```diff
@@ -548,14 +548,14 @@ private void DumpAttributes(Symbol s)
                 int j = 0;
                 foreach (var pa in sa.CommonConstructorArguments)
                 {
-                    Console.WriteLine("{0} {1} {2}", pa.ToString());
+                    Console.WriteLine(pa.ToString());
```

Comment:
It looks strange for a test to dump something to console. Perhaps, if the output isn't checked in any obvious way, we should simply remove the method.